docs/ref/migration-operations.txt

```diff
@@ -345,6 +345,12 @@ set ``atomic=False``.
     ``schema_editor.connection.alias``, where ``schema_editor`` is the second
     argument to your function).
 
+.. warning::
+
+    When migrating data across two apps, ``dependencies`` should include the target
```

Comment:
Yeah I thought about that, though I think in the majority of cases the one you need is the latest at the time you're doing the migration, or at least relying on the latest doesn't hurt anything, so we could get away with the simpler wording and not hurt anyone. But I'm ok with the longer wording too.